packages/verticaltabs/src/css/index.css

```diff
@@ -0,0 +1,311 @@
+.psds-verticaltabs {
+  height: 100%;
+  overflow-y: auto;
+  padding: var(--ps-layout-spacing-medium) 0;
+  list-style: none;
+  margin: 0;
+}
+.psds-verticaltabs__divider {
+  margin: var(--ps-layout-spacing-medium) 0;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+}
+.psds-verticaltabs__divider.psds-theme--light {
+  border-color: var(--ps-colors-border-low-on-light);
+}
+.psds-verticaltabs__divider.psds-theme--dark {
+  border-color: var(--ps-colors-border-low-on-dark);
+}
+.psds-verticaltabs__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+.psds-verticaltabs__list--collapsible {
+  overflow: hidden;
+  transition: height var(--ps-motion-speed-normal);
+  visibility: hidden;
+  -webkit-transition: height var(--ps-motion-speed-normal);
+  -moz-transition: height var(--ps-motion-speed-normal);
```

Comment:
We do not want vendor-specific attributes in the `src` css. That's why we build the css for publishing.

Reply:
Re-review: this vendor css still needs removed.